addon/templates/fallback.hbs

```diff
@@ -42,7 +42,7 @@
               {{t "fallback.sections.direct-triples.subheader"}}
             </p>
 
-            <div class="au-c-data-table__wrapper au-u-margin-bottom-large">
```

Comment:
The AuTable also has a wrapper component that provides the overflow styles, so this one shouldn't be needed. I didn't notice anything strange at least.